web/packages/teleterm/src/ui/DocumentAccessRequests/NewRequest/NewRequest.tsx

```diff
@@ -169,22 +169,39 @@ function Inner(props: { rootCluster: Cluster }) {
           />
         )}
       </StyledMain>
-      {isRequestingResourcesFromResourcesViewEnabled && (
-        <Alert kind="outline-info" mb={2}>
-          To request access to a resource, go to the{' '}
-          {/*TODO: Improve ButtonLink to look more like a text, then use it instead of the Link. */}
-          <Link
-            css={`
-              cursor: pointer;
-              color: inherit !important;
-            `}
-            onClick={openClusterDocument}
-          >
-            resources view
-          </Link>{' '}
-          or find it in the search bar.
-        </Alert>
-      )}
+      <Alert kind="outline-info" mb={2}>
+        {doesUnifiedResourcesShowBothAccesibleAndRequestableResources ? (
+          <>
+            To request access to a resource, go to the{' '}
+            {/*TODO: Improve ButtonLink to look more like a text, then use it instead of the Link. */}
+            <Link
+              css={`
+                cursor: pointer;
+                color: inherit !important;
+              `}
+              onClick={openClusterDocument}
+            >
+              resources view
+            </Link>{' '}
+            or find it in the search bar.
+          </>
+        ) : (
+          <>
+            To request access to a resource, go to the{' '}
+            {/*TODO: Improve ButtonLink to look more like a text, then use it instead of the Link. */}
+            <Link
+              css={`
+                cursor: pointer;
+                color: inherit !important;
+              `}
+              onClick={openClusterDocument}
+            >
+              resources view
+            </Link>{' '}
+            and select Access Requests &gt; Show requestable resources.
```

Comment:
Couldn't we put just the part after the link inside a conditional instead of repeating the whole block?